cirq/pasqal/pasqal_device.py

```diff
@@ -361,6 +362,22 @@ def _value_equality_values_(self) -> Any:
     def _json_dict_(self) -> Dict[str, Any]:
         return cirq.protocols.obj_to_dict_helper(self, ['control_radius', 'qubits'])
 
+    @property
+    def edges(self) -> List[Tuple['cirq.Qid', 'cirq.Qid']]:
+        """Returns a list of qubit edges on the device.
+
+        Returns:
+            All qubit pairs that are separated by a distance of 1 in a single
+            dimension.
```

Comment:
I'd change this to something like:
```suggestion
            All qubit pairs that are less than the control radius apart.
```